sk0-back/src/utils/mailer.service.ts

```diff
@@ -0,0 +1,29 @@
+import * as nodemailer from "nodemailer";
+
+export class MailerService {
+  constructor() {}
+  private transporter = nodemailer.createTransport({
+    host: "smtp.gmail.com",
+    port: 465,
+    secure: true,
+    auth: {
+      user: process.env.EMAIL_USER,
+      pass: process.env.EMAIL_PASSWORD
```

Comment:
Shouldn't we rather leverage configurations to use the credentials rather?

Reply:
I have tried to use injected ConfigService but somehow it didn't worked for me @markkovari if you have any idea it would be nice to see how 🤷‍♂️

Reply:
Marking as resolved, reading from the configs.